src/components/common/Track/index.tsx

```diff
@@ -20,8 +20,11 @@ const Track = ({ children, as: Wrapper = 'span', ...trackData }: Props): typeof
 
     const trackEl = el.current
 
-    const handleClick = () => {
-      trackEvent(trackData)
+    const handleClick = (e: MouseEvent) => {
+      // If clicked target is disabled, event.target will be trackEl
+      if (e.target !== trackEl) {
```

Comment:
What if a nested element intercepts the click? Will it always bubble up to `trackEl`?

Reply:
Yes. It seems to work nicely in my tests, even with `stopPropagation` on nested elements.